src/Yoti/Entity/Anchor.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Yoti\Entity;
+
+
+class Anchor
+{
+    /**
+     * @var string
+     */
+    protected $value;
+
+    /**
+     * @var string
+     */
+    protected $subType;
+
+    /**
+     * @var \Protobuf\Stream
+     */
+    protected $signature;
+
+    protected $artifactSignature;
+
+    /**
+     * @var \Protobuf\Stream
+     */
+    protected $signedTimeStamp;
+
+    /**
+     * @var int
+     */
+    protected $timeStamp;
+
+    /**
+     * @var array
+     */
+    protected $originServerCerts;
+
+    public function __construct(
+        $value,
+        $subType,
+        $signature,
+        $artifactSignature,
+        $signedTimeStamp,
+        array $originServerCerts
+    ) {
+        $this->value = $value;
+        $this->subType = $subType;
+        $this->signature = $signature;
+        $this->artifactSignature = $artifactSignature;
+        $this->signedTimeStamp = $signedTimeStamp;
```

Comment:
Here you need to convert this to a SignedTimestamp object. I believe this is a `\Protobuf\Stream` object, so you need to create the class from this proto https://github.com/getyoti/yoti-dotnet-sdk/blob/master/src/Yoti.Auth/ProtoBuf/common-public-api/compubapi_v1/SignedTimestamp.proto
and then create an instance using the fromStream method.